src/net/sourceforge/kolmafia/session/QuestManager.java

```diff
@@ -1682,423 +1681,469 @@ public static void updateQuestData(String responseText, String monsterName) {
+            Preferences.increment("snojoMoxieWins");
+          }
+        }
+      }
+      case "drunk cowpoke",
+          "surly gambler",
+          "wannabe gunslinger",
+          "cow cultist",
+          "hired gun",
+          "camp cook",
+          "skeletal gunslinger",
+          "restless ghost",
+          "buzzard",
+          "mountain lion",
+          "grizzled bear",
+          "diamondback rattler",
+          "coal snake",
+          "frontwinder",
+          "caugr",
+          "pyrobove",
+          "spidercow",
+          "moomy" -> {
+        Preferences.increment("lttQuestStageCount", 1);
+      }
+      case "Jeff the Fancy Skeleton",
+          "Daisy the Unclean",
+          "Pecos Dave",
+          "Pharaoh Amoon-Ra Cowtep",
+          "Snake-Eyes Glenn",
+          "Former Sheriff Dan Driscoll",
+          "unusual construct",
+          "Clara",
+          "Granny Hackleton" -> {
+        QuestDatabase.setQuestProgress(Quest.TELEGRAM, QuestDatabase.UNSTARTED);
+        Preferences.setInteger("lttQuestDifficulty", 0);
+        Preferences.setInteger("lttQuestStageCount", 0);
+        Preferences.setString("lttQuestName", "");
+      }
+      case "the ghost of Oily McBindle",
+          "boneless blobghost",
+          "the ghost of Monsieur Baguelle",
+          "The Headless Horseman",
+          "The Icewoman",
+          "The ghost of Ebenoozer Screege",
+          "The ghost of Lord Montague Spookyraven",
+          "The ghost of Vanillica \"Trashblossom\" Gorton",
+          "The ghost of Sam McGee",
+          "The ghost of Richard Cockingham",
+          "The ghost of Waldo the Carpathian",
+          "Emily Koops, a spooky lime",
+          "The ghost of Jim Unfortunato" -> {
+        QuestDatabase.setQuestProgress(Quest.GHOST, QuestDatabase.UNSTARTED);
+        Preferences.setString("ghostLocation", "");
+        ghostBusted = true;
+      }
+      case "Drab Bard", "Bob Racecar", "Racecar Bob" -> {
+        if (QuestDatabase.isQuestStep(Quest.PALINDOME, QuestDatabase.STARTED)) {
+          Preferences.increment("palindomeDudesDefeated", 1, 20, false);
+        }
+      }
+      case "fantasy bandit",
+          "fantasy ourk",
+          "fantasy forest faerie",
+          "swamp monster",
+          "cursed villager",
+          "spooky ghost",
+          "mining grobold",
+          "rubber bat",
+          "quadfaerie",
+          "druid plants",
+          "flock of every birds",
+          "plywood cultists",
+          "barrow wraith",
+          "regular thief",
+          "swamp troll",
+          "crypt creeper",
+          "\"Phoenix\"",
+          "Sewage Treatment Dragon",
+          "Duke Vampire",
+          "Spider Queen",
+          "Archwizard",
+          "Ley Incursion",
+          "Ghoul King",
+          "Ogre Chieftain",
+          "Ted Schwartz, Master Thief",
+          "Skeleton Lord" -> {
+        if (!monsterName.equals("spooky ghost")
```

Comment:
I'd probably split out the spooky ghost case for clarity

Also do we not read the monster ID? Maybe not for historical reasons but we could read it out the HTML nowadays.

Reply:
I split it out for clarity, including a comments.

Historical reasons, for sure - but as a result of those, we do not have MonsterPool.java.
That would end up being quite the substantial refactoring!

Reply:
so I don't know how `MonsterStatusTracker` works, but the final round of a fight (and every other round) includes text like:
```
<td valign="center">You're fighting <span id="monname">a Brake-Operating Trainbot</span></td><!-- MONSTERID: 2254 --></tr></tbody>
```
so I think we could extract the name (plus article) and the monster id.

I've not checked how this interacts with things like darkness.